swarmzero/server/routes/files.py

```diff
@@ -58,7 +59,13 @@ async def insert_files_to_index(files: List[UploadFile], id: str, sdk_context: S
             saved_files.append(file_path)
 
             if USE_S3:
```

Comment:
@umutunsal can you check this section

Reply:
We can add collection name to create_serverless_index(userid or session id). otherwise we will store all s3 vectors in one collection

Reply:
@umutunsal should we be using `pinecone-serverless` or `pinecone-pod`